ui/default.nginx.conf

```diff
@@ -62,7 +62,7 @@ server {
     location /healthz {
         return 200 'OK';
     }
-    location /allocation {
+    location /model/ {
```

Comment:
This was the main thing -- missing a trailing slash (h/t @nealormsbee!)

Also, we renamed it `/model/` because otherwise the URL to access the API would be `/allocation/allocation/compute`.